ironic-deployment/tls/default/tls.yaml

```diff
@@ -36,6 +36,14 @@ spec:
         - name: cert-ironic-inspector
           mountPath: "/certs/ironic-inspector"
           readOnly: true
+      - name: mariadb
+        volumeMounts:
+        - name: cert-mariadb
+          mountPath: "/certs/mariadb"
+          readOnly: true
+        - name: cert-mariadb-ca
+          mountPath: "/certs/ca/mariadb"
+          readOnly: true
```

Comment:
you need to mount the CA also in the ironic-api and ironic-conductor containers

Reply:
done!